dev-docs/bidder-adapter-1.md

```diff
@@ -358,6 +357,39 @@ function createBid(status, reqBid, response) {
 
 {% endhighlight %}
 
+#### If you would like to participate in auctions for outstream video ads
+
+Prebid.js passes a signal on video impressions that indicates whether the video context is instream or outstream.  You can start ingesting this signal if you wish.
+
+As described in [Show Outstream Video Ads]({{site.baseurl}}/dev-docs/show-outstream-video-ads.html), video ad units have a publisher-defined video context, which can be either `"instream"` or `"outstream"`:
+
+```javascript
+...
+mediaTypes: {
+    video: {
+        context: "outstream"
+    },
+},
+...
+```
+
+For an ad unit to play outstream ads, a "renderer" is required.  A renderer is a client-side media player (a JS file) that can play the VAST creative from the bid response.
+
+We currently recommend that publishers define a renderer in their ad units, as a hedge against the fact that adapters may not be providing a renderer.  Prebid.js will always use this publisher-defined renderer, if it is provided.
```

Comment:
Let's use different wording here.  Something like:

"If possible, we recommend that publishers associate a renderer with their outstream video ad units.  By doing so, all video-enabled demand partners will be able to participate in the auction, regardless of whether a given demand partner provides a renderer on its bid responses.  Prebid.js will always invoke a publisher-defined renderer on a given ad unit."